src/Codeception/Extension.php

```diff
@@ -91,10 +86,7 @@ public function _initialize(): void
         $this->_reconfigure(); // hook for BC only.
     }
 
-    /**
-     * @param string|iterable $message
-     */
-    protected function write($message): void
+    protected function write(iterable|string $message): void
```

Comment:
`iterable` was inherited from Symfony\Component\Console\Output, we don't use it anywhere, so we can simplify type to `string`.
If we keep `iterable`, we need a docblock saying that it must be an iterable of strings.

The signature of `writeln` below must match the signature of `write`.